orchagent/pfcactionhandler.cpp

```diff
@@ -412,16 +413,17 @@ PfcWdLossyHandler::~PfcWdLossyHandler(void)
 {
     SWSS_LOG_ENTER();
 
-    uint8_t pfcMask = 0;
+    uint8_t pfcTxMask = 0;
+    uint8_t pfcRxMask = 0;
 
-    if (!gPortsOrch->getPortPfc(getPort(), &pfcMask))
+    if (!gPortsOrch->getPortPfc(getPort(), &pfcTxMask, &pfcRxMask))
     {
         SWSS_LOG_ERROR("Failed to get PFC mask on port 0x%" PRIx64, getPort());
     }
 
-    pfcMask = static_cast<uint8_t>(pfcMask | (1 << getQueueId()));
+    pfcRxMask = static_cast<uint8_t>(pfcRxMask | (1 << getQueueId()));
```

Comment:
Need to update pfcTxMask as well in combined mode and in asym mode, update pfcTxMask if the queue is one specified in 'pfc_enable' fields of PORT_QOS_MAP